webapp/src/components/table/tableRow.tsx

```diff
@@ -80,6 +92,35 @@ const TableRow = (props: Props) => {
         columnRefs.set(Constants.titleColumnId, React.createRef())
     }
 
+    const handleDeleteCard = async () => {
```

Comment:
I would like to memoize this callback with `useCallback` or at least justify why not in this cases.

Reply:
Done